addons/mail/static/tests/composer/composer.test.js

```diff
@@ -420,7 +419,8 @@ test('post message on channel with "Enter" keyboard shortcut', async () => {
     await contains(".o-mail-Message");
 });
 
-test("leave command on channel", async () => {
+test.skip("leave command on channel", async () => {
+    // AKU FIXME: requires TSM fix
```

Comment:
fixme?

Reply:
Test has to be skipped so comment is relevant. However I can remove FIXME if you think it's best to remove that part of comment

Reply:
Comment to explain why it's skipped or what needs to be done are useful, but we know skip have to be fixed so just fixme is not useful.